validator/tests/unit3/test_devmode_consensus/tests.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2016 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+
+import time
+import unittest
+
+from test_journal.mock import MockStateViewFactory
+from sawtooth_validator.journal.consensus.dev_mode.dev_mode_consensus\
+    import BlockPublisher
+from sawtooth_validator.protobuf.block_pb2 import BlockHeader
+
+
+class TestCheckPublishBlock(unittest.TestCase):
+    def __init__(self, test_name):
+        super().__init__(test_name)
+
+    def create_block_header(signer_pubkey=None):
+        return BlockHeader(signer_pubkey=signer_pubkey)
+
+    def create_state_view(values):
+        state_view_factory = MockStateViewFactory(
+            {"sawtooth.consensus.min_wait_time": 0,
+             "sawtooth.consensus.max_wait_time": 0,
+             "sawtooth.consensus.valid_block_publisher": []
+             })
+        state_view = state_view_factory.create_view(None)
+        return state_view
+
+    def default_settings(self):
```

Comment:
I don't think these tests are actually executing: these methods need to start with `test_`.